styles/style.css

```diff
@@ -0,0 +1,111 @@
+
+.frame{
+	width: 460px;
+	height: 488px;
+	background: #2F4F4F;
+	border-radius: 15px;
+	margin: auto;
+}
+
+
+
+
+
+.profile{
+	float: left;
+	width: 220px;
+	background:#00BFFF;
+	margin: 17px 0 0 17px;
+	height: 452px;
+	border-radius: 15px;
+	text-align: center;}
+
+.profile p{
+	margin: 8px 0 0 0;
+	font-size: 20px;}
+	.profile img{
+		width: 110px;
+		border-radius: 50%;
+		margin: 37px 0 20px 0;}
+
+	img:hover{
+		transform: rotate(360deg);
+		transition: all 1s ease-in-out;}
+
+
+
+
+.button{
+	margin: 28px 0 0 0;}
+	.button1{
+		width:100px;
+		height: 40px;
+		border:2px solid #2F4F4F;
+		border-radius: 13px;
+		margin: 5px 0 6px 0;}
+	
+	.button2{
+		width:120px;
+		height: 40px;
+		border:2px solid #2F4F4F;
+		border-radius: 13px;}
+
+	.button3{
+		border-radius: 13px;
+		border:2px solid #2F4F4F;
+		height: 40px;
+		width: 140px;
+		margin: 6px 0 0 0;}
+
+	.button1:hover{
+		border: 4px solid #2F4F4F;
+		font-weight: bold;
+		transform: rotate(10deg);}
+
+	.button2:hover{
+		border: 4px solid #2F4F4F;
+		font-weight: bold;
+		transform: rotate(-10deg);}
+
+	.button3:hover{
+		border: 4px solid #2F4F4F;
+		font-weight: bold;
+		transform: rotate(10deg);}
+
+
+
+
+
+.statistics{
+	float: right;
+	width: 170px;
+	text-align: center;
+	margin: 0px 17px 10px 0;}
+
+	.statistics p{padding: 53px 0px 0px 0px;}
+	
+	.likes{
+	 	background: #00BFFF;
+    	height: 140px;
+   	margin-bottom: 1em;
+	 	border-radius: 15px;}
+	.likes:hover{
```

Comment:
the same with hover

.likes:hover,
.posts:hover,
.followers:hover {
common hover styles for 3 blocks
}